merged_schema/HPXMLMerged.xsd

```diff
@@ -7996,6 +8031,7 @@
 			<xs:enumeration value="crawlspace - unconditioned"/>
 			<xs:enumeration value="crawlspace - unvented"/>
 			<xs:enumeration value="crawlspace - vented"/>
+			<xs:enumeration value="manufactured home belly"/>
```

Comment:
This enumeration goes with ducts so that a duct can be _inside_ the belly. Hence the naming difference.